src/htmlParser.js

```diff
@@ -56,7 +60,8 @@ class HtmlParser {
         edited: isEdited,
         replyToken: null,
         isVerified: isVerified,
-        isOfficialArtist: isOfficialArtist
+        isOfficialArtist: isOfficialArtist,
+        customEmojis
```

Comment:
Maybe stick to current convention?
```suggestion
        customEmojis: customEmojis
```

Reply:
Good idea